activemq-broker/src/main/java/org/apache/activemq/broker/region/policy/AbstractDeadLetterStrategy.java

```diff
@@ -31,13 +31,15 @@ public abstract class AbstractDeadLetterStrategy implements DeadLetterStrategy {
     private boolean processNonPersistent = false;
     private boolean processExpired = true;
     private boolean enableAudit = true;
-    private final ActiveMQMessageAudit messageAudit = new ActiveMQMessageAudit();
```

Comment:
There's no real reason to rename this variable or change this anymore.

Reply:
Also if we decide to not have a flag for Individual and only use a message audit per destination (which I think makes more sense) then we can actually just move this message audit variable down into the Shared strategy. The abstract wouldn't need to have anything defined except the abstract method for looking up the message audit as leaving this here wouldn't make sense if it won't be used by Individual.